### PR TITLE
[MERGE after #151] Added filters to Histogram

### PR DESCRIPTION
**Pull request by r0goyal**

### CODE REVIEW

**Comment by r0goyal** [comment]
@santanusinha Review please

File: foxtrot-server/src/main/resources/console/index.htm [hunk 1]
```diff
@@ -16,880 +16,934 @@
...
+    <!--[if lt IE 9]>
+    <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
+    <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
+    <![endif]-->
+</head>
+
+<body>
+<span id="ruler"></span>
+
+<div class="container container-full">
```
Comment (santanusinha) [inline foxtrot-server/src/main/resources/console/index.htm]: Screwed up indentation

**Comment by r0goyal** [comment]
@santanusinha Review changes done

File: foxtrot-server/src/main/resources/console/js/tiles/tile.js [hunk 2]
```diff
@@ -82,6 +82,7 @@ Tile.prototype.reloadData = function () {
         console.log("did not update since query setup is not complete " + this.id);
         return;
     }
+    console.log("query : " + this.query);
```
Comment (santanusinha) [inline foxtrot-server/src/main/resources/console/js/tiles/tile.js]: Remove this

**Comment by r0goyal** [comment]
@santanusinha Change done

**Pull request merged**